cli/iconBuilder.js

```diff
@@ -0,0 +1,167 @@
+const fs = require('fs'),
+	path = require('path');
+
+const imagePath = path.join(__dirname, '../components/icons/images');
+const outputPath = path.join(__dirname, '../.generated/icons');
+
+async function getSvgsInDir(dir) {
+	return new Promise((resolve, reject) => {
+		const dirPath = path.join(imagePath, dir);
+		fs.stat(dirPath, (err, stat) => {
+			if (err) return reject(err);
+			if (!stat.isDirectory()) return resolve([]);
+			fs.readdir(dirPath, (err, files) => {
```

Comment:
If preferred, there's a synchronous version of this: `readdirSync`.

Reply:
I was trying to use the async ones as much as possible as they perform better... even though they're incredibly annoying to use haha.